src/deploy/extensions/prepare.ts

```diff
@@ -122,6 +122,7 @@ async function prepareHelper(
   if (payload.instancesToDelete.length) {
     logger.info(deploymentSummary.deletesSummary(payload.instancesToDelete));
     if (
+      !options.dryRun &&
```

Comment:
So --dry run assumes you will be deleting things? Just want to make sure, since we are thinking of changing 
--force to not delete things.. ? Not saying it's wrong, just we need to make sure that --dry-run and --force are on the same page.. whatever that is.